src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -1196,7 +1192,7 @@ private static long TZif_ToInt64(byte[] value, int startIndex)
             //
             for (int i = 0; i < t.IsStdCount && i < t.TypeCount && index < data.Length; i++)
             {
-                StandardTime[i] = (data[index++] != 0);
+                index++;
```

Comment:
Can we update the comments above this and the below loops to say "skip the Standard/GMT Time table"?

Reply:
Alternatively, we could just increment the index in one shot: `index += Math.Min(t.IsStdCount, t.TypeCount);`

Reply:
The plan was to make the review straightforward. I'm fine to rewrite more code as part of this PR.